### ISSUE TITLE
[Game-Breaking Bug] The first builder vanished and can’t be fired

### ISSUE THREAD

**Issue by Mortenkein** [ldtteam/minecolonies#1975]
### Minecolonies version

1.12.2-0.8.6623

### Expected behavior

The builder is in the colony and he’s building.

### Actual behaviour

The builder is nowhere to be found.

### Steps to reproduce the problem

1. Create a new world
2. Place the supply camp
3. Place the town hall, destroy it (one villager should appears)
4. Craft the builder’s hut, place it, choose the taiga style, and build it
5. Use the building tool to place the town hall and choose the taiga style (do not build it)
6. Place the barracks and build it
7. Upgrade the builder’s hut to level 3
8. Upgrade the barracks to level 2

Long story short : the first and only builder disapeared before completing the upgrade of the barracks, and that man cannot be fired. So, I can’t do anything anymore. I tried to recall him, I have searched for him around the colony, I waited until the night to see if he would go to the town hall. Niet, nada, he’s gone. That man abandoned the colony and nobody wants to replace him, because he’s unfirable.

**Comment by Raycoms** [comment]
2 options: find out his id in /mc citizens list and do /mc citizen respawn
id, or do  recall all in the townhall.

2018-01-07 13:22 GMT+00:00 Mortenkein <notifications@github.com>:

**Comment by Mortenkein** [comment]
I can’t use commands, because I’m in survival mode.

**Comment by Raycoms** [comment]
minecolonies commands are still available.

2018-01-07 13:28 GMT+00:00 Mortenkein <notifications@github.com>:

**Comment by Raycoms** [comment]
Or at least should.

2018-01-07 13:29 GMT+00:00 Ray Neiheiser <ray.neiheiser@gmail.com>:

**Comment by Raycoms** [comment]
Else, like I said, recall all in the townhall will restore him.

2018-01-07 13:29 GMT+00:00 Ray Neiheiser <ray.neiheiser@gmail.com>:

**Comment by Mortenkein** [comment]
Yes ! It worked. I recalled all citizens, and the builder appeared ! However, I can’t use the commands (You do not have permission to use this command). Thanks Raycoms.
